ml-algorithms/src/test/java/org/opensearch/ml/engine/httpclient/MLHttpClientFactoryTests.java

```diff
@@ -9,13 +9,16 @@
 import org.apache.http.impl.client.CloseableHttpClient;
 import org.junit.Rule;
 import org.junit.Test;
-import org.junit.rules.ExpectedException;
 
+import java.lang.reflect.InvocationTargetException;
+import java.net.Inet4Address;
 import java.net.InetAddress;
+import org.junit.rules.ExpectedException;
+
 import java.net.UnknownHostException;
 
-import static org.junit.Assert.assertNotNull;
-import static org.junit.Assert.fail;
+import java.lang.reflect.Method;
+import static org.junit.Assert.*;
```

Comment:
let's not import `.*`. https://www.baeldung.com/intellij-disable-wildcard-import

Reply:
Sorry about that, I was doing some testing and believe I accidentally left the statement in, I revised the import

Reply:
Seems like you missed the DCO again?